### PR TITLE
Abraham development

### PR DESCRIPTION
**Pull request by abrahamharos**
Hi Aki and Mark,
On this change I added two more section to the HTML and edited the CSS file.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,8 +10,16 @@
     <div id="content">
       <header>
         <h1>Abraham Haros's Portfolio</h1>
+        <img src="images/me/profilePicture.jpg" alt="Picture of me.">
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]: Can you make the alt text a little more descriptive? Where are you, are you smiling, etc.? This is important for screen reader users.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -105,60 +113,62 @@ <h3>LinkedIn: </h3>
...
+              <p>The sea at San Carlos, Sonora, Mexico.</p>
           </div>
           <div id="section">
-              <p>Two of my favortites skyscrapers at Monterrey</p>
               <img src="images/places/building1.jpg" alt="Picture of two of my favorites skyscrapers at Monterrey">
+              <p>Two of my favortites skyscrapers at Monterrey</p>
           </div>
           <div id="section">
-              <p>View of Monterrey from "Cerro de la silla"</p>
               <img src="images/places/view1.jpg" alt="A view of Monterrey from Cerro de la silla">
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]: What can you see in the view of Monterrey? Is there water, a beach, etc.? Again, these sorts of details are nice to hear if you are a screen reader user.

**Review by akihiroota87 — APPROVED** [review]
Looks good! Thanks for the upload.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (abrahamharos) [reply portfolio/src/main/webapp/index.html]: You're right, I missed that, changing it.

**Review by mschillaci — APPROVED** [review]
LGTM

**Review by akihiroota87 — APPROVED** [review]

**Pull request merged**